Nez.Portable/Utils/Screen.cs

```diff
@@ -87,6 +87,12 @@ public static bool IsFullscreen
 			set => _graphicsManager.IsFullScreen = value;
 		}
 
+		public static bool HardwareModeSwitch
```

Comment:
this needs to be wrapped in compiler define

Reply:
Done